.vscode/settings.json

```diff
@@ -1,9 +1,9 @@
 {
   "editor.defaultFormatter": "esbenp.prettier-vscode",
   "editor.formatOnSave": true,
-  "editor.codeActionsOnSave": {
-    "source.fixAll.eslint": "explicit"
-  },
+  //   "editor.codeActionsOnSave": {
```

Comment:
not sure what this does -- my vscode is slow specifically when I try to save our app. I don't know why. My macbook is way more OP than any other laptop I've had and I don't feel like our project is particularly big....